Technical-Steering-Committee/issues.md

```diff
@@ -0,0 +1,51 @@
+# Issues
+
+All the squads within the Zowe projects use GitHub to track the issues raised against the projects
+they are responsible for. The issues could be questions, bug reports, request for enhancements and others.
+Every squad responds as outlined below. 
+
+The squads can specify the details of the issue processing for specific project within the project
+repository in the ISSUES.md file in the root of the repository. 
+
+Every squad is expected to triage new issues on the regular squad meeting. The triage process should result 
+into issue categorization: bug, enhancement, question, other and either clarification or issue prioritization: critical, high, medium, low.
+
+The triaging of an issue is done by attaching labels to it (see below for the recommended set of labels):
+
+- new - The issue was not triaged yet 
+
+Types of the issues:
+
+- enhancement - Request to provide new functionality. [The recommended template](https://github.com/zowe/api-layer/blob/v2.x.x/.github/ISSUE_TEMPLATE/feature_request.md) 
```

Comment:
Add are you willing and able to work on this and help test this? This could be part of the templates.

Reply:
Added below to the engagement.